apps/frontend/src/components/ThirdParty/AccordionDisclosure.tsx

```diff
@@ -35,6 +36,36 @@ type AccordionDisclosureProps = {
 
 type Alerts = { [k: string]: DisclosureAlert }
 
+const showAbroad = (abroad: DisclosureAbroad) => {
+  if (abroad) {
+    if (abroad.abroad === true) {
+      if (abroad.refToAgreement) {
+        return <>
```

Comment:
* Ikke bruk children som en property, children er <Datatext>jsx-codeher</Datatext>
* Ikke wrap ting i <></> om du ikke må.
* if og else på `(abroad.refToAgreement)` er nesten helt like, bare forskjellig innhold på et av to felter. flytt logikk slik at vi ikke dupliserer koden